### PR TITLE
QAE-883 -  E2E test case for transparent header account icon and text color option

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Header-builder → Transparent Header → Design → Account icon and text color

### Screenshots

https://share.getcloudapp.com/WnuypKgW

### How has this been tested?

npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Transparent header settings in the customizer', () => {
+	it( 'account icon color setting should apply correctly', async () => {
+		const accountIconColor = {
+			'transparent-header-enable': 1,
+			'transparent-header-disable-latest-posts-index': 0,
+			'transparent-account-icon-color': 'rgb(96, 6, 124)',
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js:8]: it showing color-contrast error, @SimranChougule1 take a look

File: tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+						0: 'account',
+					},
+				},
+			},
+		};
+		await setCustomize( accountIconColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .ast-header-account-wrap .ast-header-account-type-icon .ahfb-svg-iconset svg circle' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js:21]: Need two selectors for SVG parts:- https://share.bsf.io/JruOJLm2 
https://share.bsf.io/2NuzL59k

File: tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js:8]: Resolved

File: tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/account-icon-text-color.test.js]: changed selector

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**